go.mod

```diff
@@ -1,33 +1,45 @@
 module github.com/google/inverting-proxy
 
-go 1.18
+go 1.24.2
```

Comment:
I'm not aware of a good reason to change this.

This isn't specifying what version of Go you use to build the packages; that's something controlled by the build environment and commands you use.

Instead, this is the *minimum* version required.

So, changing this is basically saying: "Force the build to fail if someone has anything other than the latest version of Golang installed", and that's a really heavy handed thing to do.